redis/connection.py

```diff
@@ -921,6 +924,10 @@ def __init__(
             ssl_ca_path: The path to a directory containing several CA certificates in PEM format. Defaults to None.
             ssl_password: Password for unlocking an encrypted private key. Defaults to None.
 
+            ssl_validate_ocsp: If set, perform a full ocsp validation (i.e not a stapled verification)
+            ssl_ocsp_context: A fully initialized OpenSSL.SSL.Context object to be used in verifying the ssl_ocsp_expected_cert
```

Comment:
you did not document "ssl_validate_ocsp_stapled"

Reply:
Thanks. It''s in our examples. But I've pushed this too now.